spec/features/reviews/user_can_edit_review_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
```

Comment:
We can probably call this file edit_spec.rb instead of user_can_edit_review_spec.rb